internal/commands/user/output_test.go

```diff
@@ -0,0 +1,84 @@
+package user
+
+import (
+	"testing"
+
+	"github.com/10gen/realm-cli/internal/cloud/realm"
+	"github.com/10gen/realm-cli/internal/utils/test/assert"
+)
+
+func TestUserTableHeaders(t *testing.T) {
+	for _, tc := range []struct {
+		description      string
+		authProviderType realm.AuthProviderType
+		expectedHeaders  []string
+	}{
+		{
+			description:      "should show name for apikey",
+			authProviderType: realm.AuthProviderTypeAPIKey,
+			expectedHeaders:  []string{"Name", "ID", "Type"},
+		},
+		{
+			description:      "should show email for local-userpass",
+			authProviderType: realm.AuthProviderTypeUserPassword,
+			expectedHeaders:  []string{"Email", "ID", "Type"},
+		},
+	} {
+		t.Run(tc.description, func(t *testing.T) {
+			assert.Equal(t, tc.expectedHeaders, userTableHeaders(tc.authProviderType))
+		})
+	}
+}
+
+func TestUserTableRow(t *testing.T) {
+	for _, tc := range []struct {
+		description      string
+		userEnable       bool
+		authProviderType realm.AuthProviderType
+		output           userOutput
```

Comment:
can probably get by with removing this field and adding a `userData map[string]interface{}` field for the test case struct